packages/scripts/src/create-webpack-configs.ts

```diff
@@ -212,7 +211,7 @@ export function createWebpackConfig({
         plugins: [...basePlugins, ...plugins],
         stats: 'errors-warnings',
         watchOptions: {
-            ignored: [entryTempDir],
+            ignored: Object.keys(tempEntryModules),
```

Comment:
@AviVahl  This wasn't part of the original code, but seems like ignoring watch on the entry points solves the multiple build issue. and it makes sense not to watch them because they don't actually exist

Reply:
Ignoring is fine by me, especially if it fixes the multiple builds.

TBH, I didn't know we overrode the 'watchOptions' key. we should merge it with the user's. we actually use it in several places (wcs-electron's `webpack.config.js`), so we want that picked up as well.